src/main/java/nl/hsac/fitnesse/fixture/slim/EmailFixture.java

```diff
@@ -105,25 +110,98 @@ public String ccRecipient() {
         });
     }
 
+    public ArrayList<String> listAttachments() {
+        ArrayList<String> values = null;
+        if(messageAttachments.size() > 0) {
```

Comment:
Can you format your changes so that there is a space between `if` and `for` and their opening parentheses (`(`)?